content/en/docs/reference/glossary/control-plane.md

```diff
@@ -11,3 +11,15 @@ tags:
 - fundamental
 ---
  The container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers.
+
+ <!--more--> 
+ 
+ This layer is composed by many different components, such as (but not restricted to):
+
+ * {{< glossary_tooltip text="Etcd" term_id="etcd" >}}
```

Comment:
```suggestion
 * {{< glossary_tooltip text="etcd" term_id="etcd" >}}
```

See the website for etcd to see recommended capitalization.

Reply:
Alright! Fixed.